content/docs/about/kubeflow.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "Kubeflow"
-description = "Quickly get running with your ML Workflow"
+description = "An introduction to Kubeflow"
```

Comment:
Thanks for this update @deafdave! The Kubeflow docs use title case for page titles. Please would you capitalise the word "Introduction".

Reply:
Oh wait, I'm wrong! This is the page description, not the title. :)